database_cleanup/models/purge_columns.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# © 2014-2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, fields, models
+from openerp.exceptions import UserError
+from openerp.osv import fields as legacy_fields
```

Comment:
Use `from openerp.osv import fields` show the lint error:
`database_cleanup/models/purge_columns.py:6: [W0402(deprecated-module), ] Uses of a deprecated module 'openerp.osv'`

Other good proposal is use `# pylint: disable=deprecated-module` comment in the same line.